tools/postag/src/main/scala/org/allenai/nlpstack/postag/package.scala

```diff
@@ -4,4 +4,5 @@ import org.allenai.nlpstack.core.Postagger
 
 package object postag {
   val defaultPostagger: Postagger = new FactoriePostagger
+//  val defaultPostagger: Postagger = new OpenNlpPostagger
```

Comment:
It makes it easier to switch between postaggers I guess. Will remove it none the less.